be/src/exprs/vectorized/array_functions.cpp

```diff
@@ -375,7 +378,7 @@ class ArrayContainsImpl {
         for (size_t i = 0; i < num_array; i++) {
             size_t offset = offsets_ptr[i];
             size_t array_size = offsets_ptr[i + 1] - offsets_ptr[i];
-            uint8_t found = 0;
+            size_t found = 0;
```

Comment:
It's better to name it position? And turn it to bool when assign it to result?

Reply:
I add a new local var to indicate the postion. Will it be fine?